swarmform/scripts/sform_run.py

```diff
@@ -0,0 +1,200 @@
+            err_message += ' Type "lpad init" if you would like to set up a file that specifies ' \
+                           'location and credentials of your Mongo database (otherwise use default ' \
+                           'localhost configuration).'
+        raise ValueError(err_message)
+
+
+def reset(args):
+    sp = get_sp(args)
+    if not args.password:
+        if input('Are you sure? This will RESET {} workflows and all data. (Y/N)'.format(
+                sp.workflows.count()))[0].upper() == 'Y':
+            args.password = datetime.datetime.now().strftime('%Y-%m-%d')
+        else:
+            raise ValueError('Operation aborted by user.')
+    sp.reset(args.password)
+
+
+def add_sf(args):
+    sp = get_sp(args)
+    if args.dir:
+        files = []
+        for f in args.sf_file:
+            files.extend([os.path.join(f, i) for i in os.listdir(f)])
+    else:
+        files = args.sf_file
+    for f in files:
+        fwf = SwarmFlow.from_file(f)
+        if args.check:
+            from fireworks.utilities.dagflow import DAGFlow
+            DAGFlow.from_fireworks(fwf)
+        sp.add_sf(fwf)
+
+
+def get_sf(args):
+    if args.id:
+        get_sf_by_id(args)
+    if args.name:
+        get_sf_by_name(args)
+
+
+def get_sf_by_id(args):
+    sp = get_sp(args)
+
+    if isinstance(args.sf_id, int):
+        swarmflow = sp.get_sf_by_id(args.sf_id)
+        args.outupt(swarmflow)
+    else:
+        raise ValueError('Incorrect id type {}.'.format(type(args.sf_id)))
+
+
+def get_sf_by_name(args):
+    sp = get_sp(args)
+    swarmflow = sp.get_sf_by_name(args.sf_name)
+    args.outupt(swarmflow)
+
+
+# Cluster the jobs in a workflow
+def cluster_workflow(args):
+    sp = get_sp(args)
+    unclustered_sf = sp.get_sf_by_id(args.sf_id)
+    unclustered_sf_fw_id = unclustered_sf.fws[0].fw_id
+    clustered_workflow= cluster_sf(sp, args.sf_id)
+    sp.add_sf(clustered_workflow)
+    sp.archive_wf(unclustered_sf_fw_id)
+    sp.m_logger.info('Workflow with id {} clustered succesfully'.format(args.sf_id))
+
+
+def sform():
+    m_description = 'A command line interface to SwarmForm. For more help on a specific command, ' \
+                    'type "sform <command> -h".'
+
+    parser = ArgumentParser(description=m_description)
+    parser.add_argument('-o', '--output', choices=["json", "yaml"],
+                        default='json', type=lambda s: s.lower(),
+                        help='Set output display format to either json or YAML. '
+                             'YAML is easier to read for long documents. JSON is the default.')
+
+    parser.add_argument('-l', '--launchpad_file', help='path to LaunchPad file containing '
+                                                       'central DB connection info',
+                        default=None)
+    parser.add_argument('-c', '--config_dir',
+                        help='path to a directory containing the LaunchPad file (used if -l unspecified)',
```

Comment:
Can we change here _launchpad_ and _LaunchPad_ to _swarmpad_ or is there any problem with that?

Reply:
I tried changing that and faced many issues. Better keep it this way until we want to make this production ready